doc/content/integrations/webhooks/creating-webhooks.md

```diff
@@ -14,18 +14,30 @@ In addition to the written instructions linked below, a video with instructions
 {{< youtube "trNIzIf3TZo" >}}
 </details>
 
-Creating a webhook requires you to have an HTTP(S) endpoint available.
+Creating a webhook requires you to have an HTTP(S) endpoint available. If you don't have an endpoint available for testing, you can test with a free service like [Webhook.site](https://webhook.site).
 
-In your application select the **Webhooks** submenu from the **Integrations** side menu. Clicking on the **+ Add Webhook** button will open the Webhook creation screen. 
+In your application select the **Webhooks** submenu from the **Integrations** side menu. Clicking on the **+ Add Webhook** button will open the list of Webhook templates. 
 
-Fill in your **Webhook ID**, choose **Webhook Format** and provide **Base URL**.
+{{< figure src="../webhook-templates.png" alt="Webhook templates" >}}
 
-{{< figure src="../webhook-creation.png" alt="Webhook creation screen" >}}
+To use a predefined webhook template, choose it. Otherwise, to create your own custom webhook, choose **Custom webhook** and follow the instructions below.
 
-The paths are appended to the base URL. For example, if you follow the example in the picture above, the Application Server will perform `POST` requests on the endpoint `https://app.example.com/lorahooks/join` for join-accepts and `https://app.example.com/lorahooks/up` for uplink messages.
+{{< figure src="../add-webhook.png" alt="Webhook creation screen" >}}
+
+Give your webhook a **Webhook ID** so that you can identify it. Choose JSON as the **Webhook Format**. For **Base URL**, provide the endpoint URL that the webhook should call, i.e the URL of your webhook integration.
+
+The **Downlink API key** field allows you to provide an API key which will be sent with all messages to the webhook endpoint. This can be useful to allow your integration to schedule downlinks without requiring you to hardcode an API key in the endpoint.
+
+{{< note >}}
+Your integration may also [schedule downlinks using an API key with appropriate rights]({{< relref "scheduling-downlinks" >}}), regardless of the value in the **Downlink API key** field.
```

Comment:
Not sure about this wording "integration may schedule downlinks". I would say the integration is not the one who schedules them, but the one who enables them to be scheduled